common/djangoapps/student/forms.py

```diff
@@ -180,7 +182,8 @@ def save(self,  # pylint: disable=arguments-differ
         user.
         """
         for user in self.users_cache:
-            send_account_recovery_email_for_user(user, request, user.account_recovery.secondary_email)
+            if user_belongs_to_edly_organization(request):
```

Comment:
why user is passed into  `user_belongs_to_edly_organization` here?

Reply:
we need to check if the current user for which the `password-reset` request is initiated is linked with the current site.

Reply:
sorry i have missed a code while cleaning updating shortly